src/pages/[platform]/build-a-backend/storage/list-files/index.mdx

```diff
@@ -109,6 +109,43 @@ function processStorageList(response) {
 
 This places each item's data inside a special `__data` key.
 
+### Exclude Subpaths
+
+When objects are organized using a prefix you can return only those objects within the `path` you've provided, and not those within nested subpaths.
```

Comment:
this is a bit unclear. i think we need to explain the concept of subpath here. since we are newly introducing it.

Reply:
Yeah basically a `subpath` is a common `path` between the main `path` and a `delimiter`.